test/Twilio.Test/Http/SystemNetHttpClientTest.cs

```diff
@@ -239,8 +238,10 @@ public void TestMakeRequestAddsHeadersAndUserAgent()
             Assert.AreEqual("utf-8", internalRequest.Headers.AcceptEncoding.ToString());
 
             Assert.IsNotNull(internalRequest.Headers.UserAgent);
-            Regex rgx = new Regex(@"^twilio-csharp/[0-9.]+(-rc\.[0-9]+)?\s\(\w+\s\w+\)\s[.\s\w]+/[^\s]+$");
-            Assert.IsTrue(rgx.IsMatch(internalRequest.Headers.UserAgent.ToString()));
+            Assert.That(
```

Comment:
By using the native regex assertion, the string and regex will be printed to the console for easier debugging when failing.